openfl/component/aggregator/aggregator.py

```diff
@@ -117,9 +117,11 @@ def __init__(
             # Cleaner solution?
             self.single_col_cert_common_name = ""
 
-        self.straggler_handling_policy = (
-            straggler_handling_policy or CutoffTimeBasedStragglerHandling()
-        )
+        if straggler_handling_policy == CutoffPolicy:
+            self.straggler_handling_policy = straggler_handling_policy()
+        else:
+            self.straggler_handling_policy = straggler_handling_policy
+
```

Comment:
I suggest creating a `DefaultPolicy`. This simplifies:

1. `self.straggler_handling_policy = straggler_handling_policy()` regardless of what is given, saves 4 lines.
2. `DefaultPolicy` would be the equivalent of providing no straggler handling capabilities. Dummy class.

Also the condition check is wrong. What if `PercentagePolicy` is provided? It will go to else block and fail in code, it seems.

Reply:
Incorporated the suggestion you provided.